flask/cli.py

```diff
@@ -37,19 +37,46 @@ def find_best_app(module):
     # Search for the most common names first.
     for attr_name in 'app', 'application':
         app = getattr(module, attr_name, None)
-        if app is not None and isinstance(app, Flask):
+        if isinstance(app, Flask):
             return app
 
     # Otherwise find the only object that is a Flask instance.
-    matches = [v for k, v in iteritems(module.__dict__)
-               if isinstance(v, Flask)]
+    matches = [
```

Comment:
personally i'd just put all in one line and screw the dumb 79-char  line length :p